ui/static/ui/js/utils.jsx

```diff
@@ -109,8 +109,13 @@ define("utils", ["jquery", "lodash", "reactaddons", "select2"],
       //this function can be used only if the component has been mounted
       var thiz = this;
       var isMultiTerms = this.props.multiple;
+
       //not allowing the clear for the multi terms dropdown
-      var allowClear = !isMultiTerms;
+      var allowClear;
+      if (this.props.allowClear === undefined) {
```

Comment:
`this.props.allowClear` is only being tested against `undefined`. What if the caller passes `allowClear={true}`?